app/src/main/java/com/nytimes/android/sample/SampleApp.java

```diff
@@ -57,11 +61,23 @@ private void initPersister() {
 
     private Store<RedditData, BarCode> provideRedditStore() {
         return StoreBuilder.<RedditData>barcode()
-                .fetcher(barCode -> provideRetrofit().fetchSubreddit(barCode.getKey(), "10"))
-                .memory(CacheBuilder.newBuilder()
-                        .maximumSize(1)
-                        .expireAfterWrite(10, TimeUnit.SECONDS)
-                        .build())
+                .fetcher(new Fetcher<RedditData, BarCode>() {
+                    @Nonnull
+                    @Override
+                    public Observable<RedditData> fetch(@Nonnull BarCode barCode) {
+                        Log.d("fetcher", "Going for fetcher");
+                        return provideRetrofit().fetchSubreddit(barCode.getKey(), "10");
+                    }
+                })
+//                .fetcher(barCode -> provideRetrofit().fetchSubreddit(barCode.getKey(), "10"))
+                .memoryPolicy(
+                    MemoryPolicy
```

Comment:
Let's simplify this to MemoryPolicy.builder() which then returns an instance of the new builder.